Automattic/pocket-casts-ios
# Allows feed URL only podcasts to be imported from the Apple Podcasts shortcut

Fixes #632

This fixes podcasts without a store URL being ignored by the import shortcut. This also updates the shortcut install link to a dynamic link so we don't need to push out a new update in order to release a "new version" of the shortcut


## To test

1. Open Apple Podcasts
2. Add a podcast that's not available on Apple
   - You can use my test one if you don't have one: https://podcast.emily.tools/
3. Go to https://pocketcasts.com/import-from-apple-podcasts/ on your iPhone
4. Install the shortcut
5. Tap it to run it
6. ✅ Verify the export opens the app
7. ✅ Verify it imports successfully
8. ✅ Verify your are subscribed to the test podcast

## Checklist
